FRENCH/src/ch13-01-closures.md

```diff
@@ -104,12 +104,12 @@ app’s frontend isn’t relevant to the use of closures, we’ll hardcode value
 representing inputs to our program and print the outputs.
 -->
 
-Ensuite, nous avons la fonction `main` qui contient les parties importantes de
-cet exemple d'application d'entraînement. Cette fonction représente le code que
-l'application appellera lorsqu'un utilisateur demande un plan d'entraînement.
-Comme l'interaction avec l'application dorsale n'est pas adapté à l'utilisation
-des fermetures, nous allons coder en dur les valeurs représentant les entrées de
-notre programme, puis afficher les résultats.
+Ensuite, nous avons la fonction `main` qui contient les parties de l'application
+d'entraînement qui sont importantes pour cet exemple. Cette fonction représente le code que
+l'application appellera lorsqu'un utilisateur demande un programme d'entraînement.
+Comme l'interaction avec l'interface de l'application n'apporte rien dans l'étude 
+de l'utilisation des fermetures qui nous occupe ici, nous allons nous contenter de coder 
+en dur les valeurs représentant les entrées de notre programme puis afficher les résultats obtenus.
```

Comment:
```suggestion
Comme l'interaction avec l'application frontale n'a pas d'intérêt dans notre
étude de l'utilisation des fermetures, nous allons nous contenter de coder en
dur les valeurs représentant les entrées de notre programme puis afficher les
résultats obtenus.
```

Reply:
app's frontend in my opinion is the part of the application the user interacts with ... maybe "interface" is not the best way to reflect the intent but I think "application frontal" is not accurate. Maybe simply "interaction avec l'application" is closer ?

Reply:
This translation term was build from an Wikipedia article like https://fr.wikipedia.org/wiki/Frontal_(serveur)

I agree with `interface frontale` change, but in this case we need to change it in `translation-terms.md` and all other pages which uses this term 😉

Reply:
Looks like `frontend` is only used in this part of the book so it makes this change easy.

```suggestion
Comme l'interaction avec l'interface frontale de l'application n'apporte rien dans l'étude 
de l'utilisation des fermetures qui nous occupe ici, nous allons nous contenter de coder 
en dur les valeurs représentant les entrées de notre programme puis afficher les résultats obtenus.
```